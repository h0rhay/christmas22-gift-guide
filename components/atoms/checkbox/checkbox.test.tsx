import { render } from '@testing-library/react';
import Checkbox from '.';
import { ThemeProvider } from 'styled-components';
import Theme from 'styles/theme';

const renderComponent = () =>
  render(
    <ThemeProvider theme={Theme}>
      <Checkbox label='foobar' id='foo' />
    </ThemeProvider>
  );

describe('Checkbox', () => {
  it('should match snapshot', () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });
});
