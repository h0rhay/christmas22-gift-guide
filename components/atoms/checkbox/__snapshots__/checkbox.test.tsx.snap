// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should match snapshot 1`] = `
<div>
  <div
    class="checkbox__CheckboxContainer-sc-tki452-0 gfYtLH"
  >
    <label
      class="container"
      for="foo"
    >
      <input
        id="foo"
        type="checkbox"
      />
      <span
        class="checkmark"
      />
      foobar
    </label>
  </div>
</div>
`;
